mm2src/mm2_bitcoin/script/src/sign.rs

```diff
@@ -207,6 +210,7 @@ impl From<TransactionInputSigner> for Transaction {
                     previous_output: input.previous_output,
                     script_sig: vec![].into(),
                     sequence: input.sequence,
+                    // FIXME: Why not add the witness from the unsigned transaction input? optimization?
```

Comment:
it looks like that 'witness' field in UnsignedTransactionInput is not used specifically for signing. Maybe it could be eliminated